### DISCUSSION TITLE
Windows

### DISCUSSION THREAD

**Discussion by Jichao96** [insarlab/MintPy#695] [Q&A] [answered]
I would like to know whether MintPy can be run under Windows. Thanks.

**Comment by jhkennedy** [comment, marked as answer]
**Edit:** MintyPy is available on windows via conda-forge as of MintPy v1.3.2, and all *known* issues have been resolved.

---

@Jichao96 I would say MintPy on windows is still experimental. 
* PySolid is now available on windows via conda-forge (insarlab/PySolid#19) but it's likely that there are issues on windows still (insarlab/PySolid#15)
* Instructions for building MintPy on windows was added in #647, and some issues with numpy on windows were resolved there, but it hasn't been thoroughly tested. 

Next release, we should be able to provide windows builds for Mintpy on conda-forge, but they will likely still be experimental at that point. 

If you're interested in trying out the experimental version, let me know! I can provide updated instructions beyond what's in the installation guide since windows support is being actively developed.

**Comment by hardreddata** [comment]
You can run MintPy using Windows Subsystem for Linux https://docs.microsoft.com/en-us/windows/wsl/install

**Comment by pbrotoisworo** [comment]
I'm a Windows user and it works for me. I use it in the command prompt. I'm not using Windows Subsystem for Linux.
